src/components/heroSection.css

```diff
@@ -1,60 +1,60 @@
 video {
-    object-fit: cover;
-    width: 100%;
-    height: 100%;
-    position: fixed;
-    z-index: -1;
+  object-fit: cover;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  z-index: -1;
 }
 
 .hero-container {
-    /* background: url('../../assets/images/img-home.jpg') center center/cover no-repeat; */
-    height: 100vh;
-    width: 100%;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
-    object-fit: contain;
+  /* background: url('../../assets/images/img-home.jpg') center center/cover no-repeat; */
+  height: 100vh;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  object-fit: contain;
 }
 
 .hero-container > h2 {
-    color: aliceblue;
-    font-size: 100px;
+  color: aliceblue;
+  font-size: 100px;
 }
 
 .hero-container > p {
-    color: aliceblue;
-    font-size: 32px;
+  color: aliceblue;
```

Comment:
Do all the colors match yet? Because you were using color variables but here and there I see you're not.